contracts/mocks/ForeignBridgeErcToNativeMock.sol

```diff
@@ -17,4 +17,14 @@ contract ForeignBridgeErcToNativeMock is ForeignBridgeErcToNative {
         // Address generated in unit test
         return ISaiTop(0x96bc48adACdB60E6536E55a6727919a397F14540);
     }
+
+    bytes32 internal constant CHAI_TOKEN_MOCK = 0x5d6f4e61a116947624416975e8d26d4aff8f32e21ea6308dfa35cee98ed41fd8; // keccak256(abi.encodePacked("chaiTokenMock"))
+
+    function setChaiToken(IChai _chaiToken) external {
+        addressStorage[CHAI_TOKEN_MOCK] = _chaiToken;
+    }
+
+    function chaiToken() public view returns (IChai) {
```

Comment:
Does it make sense to redefine `uintStorage[INTEREST_COLLECTION_PERIOD]` in this mock as well in order to cover the following functionality by tests: the paying interest is possible after `INTEREST_COLLECTION_PERIOD` expiration?

Reply:
Yes, this makes sense.

Reply:
Used `setInterestCollectionPeriod()` for testing this case.